airbyte-integrations/connectors/source-linkedin-ads/integration_tests/abnormal_state.json

```diff
@@ -2,106 +2,256 @@
   {
     "type": "STREAM",
     "stream": {
-      "stream_state": { "lastModified": "2050-01-01" },
-      "stream_descriptor": { "name": "account_users" }
+      "stream_descriptor": {
```

Comment:
No, all basic streams affected as they use per partition state.

Reply:
What happens if we use a state like `{ "lastModified": "2024-01-01" }` with the low-code version?

Reply:
We'll receive an error as the cursor key couldn't be found in the state.

Reply:
Ok! I think I originally misunderstood this because I didn't get that all the streams were affected. I guess I missed the migration guide update. Now things make sense. Thanks Serhii!